odoo/modules/module.py

```diff
@@ -8,15 +8,16 @@
 import importlib
 import logging
 import os
-import pkg_resources
 import re
 import sys
 import warnings
-from os.path import join as opj, normpath
+from os.path import join as opj
+from os.path import normpath
 
 import odoo
-import odoo.tools as tools
 import odoo.release as release
+import odoo.tools as tools
+import pkg_resources
```

Comment:
I know the imports are ill-sorted but our policy is to sort them only when we actually add/remove new imports for better compatibility with our @fw-bot, can you please restore the original code?

Reply:
Yes indeed, my IDE changed them automatically.